content/en/developers/guide/dogshell.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Dogshell
+description: "Use Datadog's API from Terminal or Shell"
+aliases:
+  - /developers/faq/dogshell-quickly-use-datadog-s-api-from-terminal-shell
+  - /developers/guide/dogshell-quickly-use-datadog-s-api-from-terminal-shell
+---
+
+You can use the Datadog API on the command line using a wrapper called Dogshell.
+
+## Install Dogshell
+
+Dogshell comes with the officially supported [datadogpy Python library][1], which is often used to send data to Datadog with [DogStatsD][2]. To install the library with PIP, run the following command:
+
+{{< code-block lang="shell" >}}
+pip install datadog
+{{< /code-block >}}
+
+Depending on your environment, you might have to add the library to your PATH. See the [datadogpy GitHub repo][3] for alternate installation instructions.
+
+## Configure Dogshell
+
+Dogshell uses a configuration file called `.dogrc` to store your API key, application key, and Datadog site.
+
+To configure Dogshell:
+1. Create a `.dogrc` file in your home directory:
+   {{< code-block lang="shell" >}}
+touch ~/.dogrc
+{{< /code-block >}}
+
+1. Add the following content to the file, replacing `DATADOG_API_KEY` and `YOUR_APPLICATION_KEY` with your API key and application key respectively:
+   ```conf
+   [Connection]
+   apikey = <DATADOG_API_KEY>
+   appkey = <YOUR_APPLICATION_KEY>
```

Comment:
Don't we usually use DD_API_KEY and DD_APP_KEY?

Reply:
```suggestion
   apikey = DD_API_KEY
   appkey = DD_APP_KEY
```